ci/docker/python-wheel-windows-test-vs2019.dockerfile

```diff
@@ -40,8 +40,9 @@ ARG python=3.8
 RUN (if "%python%"=="3.8" setx PYTHON_VERSION "3.8.10" && setx PATH "%PATH%;C:\Python38;C:\Python38\Scripts") & \
     (if "%python%"=="3.9" setx PYTHON_VERSION "3.9.13" && setx PATH "%PATH%;C:\Python39;C:\Python39\Scripts") & \
     (if "%python%"=="3.10" setx PYTHON_VERSION "3.10.11" && setx PATH "%PATH%;C:\Python310;C:\Python310\Scripts") & \
-    (if "%python%"=="3.11" setx PYTHON_VERSION "3.11.5" && setx PATH "%PATH%;C:\Python311;C:\Python311\Scripts") & \
-    (if "%python%"=="3.12" setx PYTHON_VERSION "3.12.0" && setx PATH "%PATH%;C:\Python312;C:\Python312\Scripts")
+    (if "%python%"=="3.11" setx PYTHON_VERSION "3.11.9" && setx PATH "%PATH%;C:\Python311;C:\Python311\Scripts") & \
+    (if "%python%"=="3.12" setx PYTHON_VERSION "3.12.4" && setx PATH "%PATH%;C:\Python312;C:\Python312\Scripts") & \
+    (if "%python%"=="3.13" setx PYTHON_VERSION "3.13.0-rc1" && setx PATH "%PATH%;C:\Python313;C:\Python313\Scripts")
 
 # Install archiver to extract xz archives
 RUN choco install -r -y --no-progress python --version=%PYTHON_VERSION% & \
```

Comment:
This is still failing with `Unable to resolve dependency 'python3 (= 3.13.0-rc1)'.` (although the version is certainly available at https://community.chocolatey.org/packages/python)

Reply:
Hmm, it seems like this is due to `--pre` missing from the `choco install` invocation. Is it okay to add that unconditionally?

Reply:
Worth a try (not familiar with chocolatey ..)

Reply:
yeah, that sounds good, I added the `--pre` when I was testing 3.12 in the past